sphinx/screens/chats/chat-common/chat-common/src/main/java/chat/sphinx/chat_common/ui/viewstate/messageholder/MessageHolderViewState.kt

```diff
@@ -291,6 +291,10 @@ internal sealed class MessageHolderViewState(
                 list.add(MenuItemState.Boost)
             }
 
+            if (message.isMediaAttachment) {
+                list.add(MenuItemState.SaveFile)
```

Comment:
should we add this as the implementation in the VM notify's the user "TODO not implemented" ??